foyer/forcefields/oplsaa.xml

```diff
@@ -182,15 +182,15 @@
   <Type name="opls_183" class="opls_183" element="C" mass="12.011"/>
   <Type name="opls_184" class="opls_184" element="C" mass="12.011"/>
   <Type name="opls_185" class="HC" element="C" mass="1.00800"   def="HC[O;%opls_180]" desc="H(COR), alpha H ether" overrides="opls_140" doi="10.1021/ja9621760"/>
-  <Type name="opls_186" class="opls_186" element="O" mass="15.9994"/>
+  <Type name="opls_186" class="opls_186" element="O" mass="15.9994" def="[O;X2]([C;%opls_193](O)(C)H)" desc="O: acetal ether" overrides="opls_180"/>
```

Comment:
The outer parentheses in this SMARTS string are redundant. I think this can safely be written as `[O;X2][C;%opls_193](O)(C)H`.